www/less/widenImages.less

```diff
@@ -4,31 +4,9 @@
     display: block;
     margin-left: auto;
     margin-right: auto;
-}
-
-@media (orientation: portrait) {
-    .wideImageOverride {
```

Comment:
wideImageOverride isn't removed, just the @ media variants. I forgot about the landscape phone (was thinking it was portrait only) I'll update so that it remains at 50% for the landscape phone.